.github/workflows/build-test.yml

```diff
@@ -30,7 +30,7 @@ jobs:
           if ! [ "${VER_DIFF}" ]; then
             bash update_version.sh --commit && git push
           fi
-          CL_DIFF=$(git diff ${{ github.base_ref }} ${{ github.sha }} -- ./CHANGELOG.md)
+          CL_DIFF=$(git diff ${{ github.event.pull_request.base.sha }} ${{ github.sha }} -- ./CHANGELOG.md)
```

Comment:
I saw you were still making changes so I held off my comment, but I was going to say switching to `.base_ref` seemed unnecessary when `event.pull_request.base.sha` was working fine before ;)

Reply:
Hehehe